packages/nx/src/native/tests/workspace_files.spec.ts

```diff
@@ -152,30 +148,7 @@ describe('workspace files', () => {
     );
 
     expect(globalFiles).toEqual([]);
-    expect(projectFileMap['repo-name']).toMatchInlineSnapshot(`
-      [
-        {
-          "file": "jest.config.js",
-          "hash": "3244421341483603138",
-        },
-        {
-          "file": "nx.json",
-          "hash": "1389868326933519382",
-        },
-        {
-          "file": "package.json",
-          "hash": "14409636362330144230",
-        },
-        {
-          "file": "project.json",
-          "hash": "4357927788053707201",
-        },
-        {
-          "file": "src/index.js",
-          "hash": "3244421341483603138",
-        },
-      ]
-    `);
+    expect(projectFileMap['repo-name']).toMatchInlineSnapshot(`undefined`);
```

Comment:
This should be `projectFileMap['.']`. The undefined in the snapshot isn't right 😅